hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/AcceptanceTestProperties.java

```
@@ -87,6 +94,22 @@ public class AcceptanceTestProperties {
     @NotNull
     private NodeNameEnum nodeType = NodeNameEnum.MIRROR;
 
+    @PostConstruct
+    public void configureOperator() throws Exception {
+        var configuredOperator = AccountId.fromString(operatorId);
+        if (configuredOperator.realm != commonProperties.getRealm()
+                || configuredOperator.shard != commonProperties.getShard()) {
+            if (DEFAULT_OPERATOR_ID.equals(operatorId)) {
+                operatorId = String.format(
+                        "%d.%d.%d", commonProperties.getShard(), commonProperties.getRealm(), configuredOperator.num);
+            } else {
+                throw new IllegalArgumentException(String.format(
+                        "Operator account %s must be in shard %d and realm %d",
+                        operatorId, commonProperties.getShard(), commonProperties.getRealm()));
```

Might be cleaner to fail fast so you can remove the extra indentation.